content/docs/guides/logical-replication-neon.md

```diff
@@ -337,14 +341,27 @@ If you require different values for these parameters, please contact Neon suppor
 
 ### Unused replication slots
 
-Neon removes _unused_ replication slots after a period of time to avoid unnecessary retention of write-ahead logs, which prevents removing data snapshots that are no longer required.
+To prevent storage bloat, **Neon automatically removes an _inactive_ replication slot if you have other _active_ replication slots**. Removal occurs after 75 minutes. 
+
+If you have only one replication slot, and that slot becomes inactive, it is not removed due to inactivity because a single replication slot does not create bloat. If you find that your single replication slot has been removed, please contact [Neon Support](/docs/introduction/support) for remediation.
+
+#### What causes a replication slot to become inactive? 
+
+An _inactive_ replication slot is often the result of a _dead subscriber_, where the replication slot is not dropped after a subscriber is deactivated or becomes unavailable. An inactive replication slot can also result from a replication delay configured on the subscriber. For example, some subscribers allow you to configure the frequency of replication or set a replication delay to minimize usage.
```

Comment:
Let's start this paragraph with ''An _inactive_ replication slot is the one which doesn't acknowledge progress on flush_lsn (as reported by same named field in pg_stat_replication)" for a long time.

Reply:
Fixed.